src/cmd/sql_cmd.h

```diff
@@ -141,14 +141,53 @@ std::string ExecFetch(const std::string& sql) {
     return ss.str();
 }
 
-void HandleSQL(const std::string& sql) {
-    std::cout << ExecFetch(sql);
+void HandleSQL(const std::string& sql) { std::cout << ExecFetch(sql); }
+
+std::string SafeGetString(std::shared_ptr<hybridse::sdk::ResultSet> rs, int idx) {
+    std::string tmp;
+    if (!rs->GetString(idx, &tmp)) {
+        LOG(WARNING) << "fail to get string in col " << idx;
+        return "";
+    }
+    return tmp;
+}
+
+void CheckAllTableStatus() {
+    hybridse::sdk::Status status;
+    auto rs = sr->ExecuteSQL("show table status like '%'", &status);
+    if (status.IsOK()) {
+        // ref GetTableStatusSchema, just use idx to get column 8, 13
+        while (rs->Next()) {
+            auto table = SafeGetString(rs, 1);
+            auto db = SafeGetString(rs, 2);
+            auto pu = SafeGetString(rs, 8);
+            auto warnings = SafeGetString(rs, 13);
+            std::string msg = absl::StrCat("table ", db, ".", table, " is broken, `show table status` to check detail");
+            bool is_broken = false;
+            if (pu != "0") {
+                is_broken = true;
+                msg.append(", unalive partition: ").append(pu);
+            }
+            if (!warnings.empty()) {
+                is_broken = true;
+                msg.append(", warning preview: ").append(warnings.substr(0, 100));
+            }
+            if (is_broken) {
+                LOG(WARNING) << msg;
+            }
+        }
+    } else {
+        LOG(ERROR) << "fail to get all table status, " << status.ToString();
+    }
 }
 
 // cluster mode if zk_cluster is not empty, otherwise standalone mode
 void Shell() {
     DCHECK(cs);
     DCHECK(sr);
+    // before all, check all table status
+    CheckAllTableStatus();
```

Comment:
add hint, openmldb_tool inspect can get full and clear report

Reply:
![image](https://github.com/4paradigm/OpenMLDB/assets/24697960/13789352-bc90-404e-a1b7-926055abb5c4)